content/en/docs/reference/command-line-tools-reference/feature-gates/dra-admin-access.md

```diff
@@ -0,0 +1,18 @@
+---
+title: DRAAdminAccess
+content_type: feature_gate
+_build:
+  list: never
+  render: false
+
+stages:
+  - stage: alpha
+    defaultValue: false
+    fromVersion: "1.32"
+---
+Enables support for requesting [admin access](/docs/concepts/scheduling-eviction/dynamic-resource-allocation/#admin-access)
+in a ResourceClaim. A ResourceClaim
+with admin access grants access to devices which are in use and may enable
+additional access permissions when making the device available in a container.
+
+Has no effect unless DynamicResourceAllocation is also enabled.
```

Comment:
Maybe:
```suggestion
This feature gate has no effect unless you also enable the `DynamicResourceAllocation` feature gate.
```
Extra credit: put a comment in DynamicResourceAllocation to update this file when graduating `DynamicResourceAllocation` to stable.

Reply:
Done (both).